augly/image/functional.py

```diff
@@ -1786,3 +1786,56 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 5.0,
+    max_level: float = 10.0,
+    axis: str = "x_axis",
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Image.Image:
+    
+    """
+    Skew image with respect to X-axis or Y-axis
+    
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+        
+    @param level: Level of the operation that will be in [0, `PARAMETER_MAX`]
+    
+    @param max_level: Maximum value that the operation can have. This will be
+        scaled to level / PARAMETER_MAX.
+        
+    @param axis: Axis with respect to change like x-axis or y-axis
+                
+    @param output_path: the path in which the resulting image will be stored.
+        If None, the resulting PIL Image will still be returned
+
+    @param metadata: if set to be a list, metadata about the function execution
+        including its name, the source & dest width, height, etc. will be appended
+        to the inputted list. If set to None, no metadata will be appended or returned
+
+    @returns: the augmented PIL Image
+    """
+    func_kwargs = imutils.get_func_kwargs(metadata, locals())
+    image = imutils.validate_and_load_image(image)
+    
+    level = float(np.random.uniform(low=0.1, high=level)) * max_level / 10
```

Comment:
What does this do? Why are you sampling a `level` at all when we have the `level` arg passed in? Why multiple by `max_level / 10`?

Reply:
Because level when see user give we have to make first float and after sampling it will never give error or we can say we are going to normalize our level.
See, if user give level : 1.0 and Max_level : 10.0
then after sampling it will become  0.9010069136062707 , it is same as like normalizations.

Reply:
I'm not sure what you mean by this. Can we get rid of this line if the user just passes in a `level` that is in the correct range? Can you try running the augmentation to find out what would be a correct range for `level` so we wouldn't need this confusing sampling?